packages/server/src/storage.ts

```diff
@@ -36,3 +39,73 @@ export class LevelDBStorage implements Storage {
         throw "not implemented";
     }
 }
+
+// Add support for AWS DynamoDB
+// Limitations with DynamoDB is that Total Item Size is only 400KB
+// This means that a Vault can not exceed 400KB In size with the current
+// Storage Implimentation as is. That's about 500 Standard Items
+export class DynamoDBStorage implements Storage {
+    private _db: any;
+    private table: string;
+
+    constructor(public credentials: CredentialProviderChain, region: AwsRegion, table: string) {
+        this._db = new DynamoDB({credentialProvider: credentials, region: region});
+        this.table = table;
+    }
+
+    async get<T extends Storable>(cls: StorableConstructor<T> | T, id: string) {    
+        try {
+            const res = cls instanceof Storable ? cls : new cls();
+            const raw = await this._db.getItem({
+                Key: {
+                    "storeable": {
+                      S: `${res.kind}_${id}`
+                     }
+                   }, 
+                   TableName: this.table
+            }).promise();
+            
+            // DynamoDB does not return an error on empty result.
+            // Checking whether the _item_ property is defined to
+            // determine whether the Item is not found.
+
+            if( typeof(raw.Item) === 'undefined') {
+                throw new Err(ErrorCode.NOT_FOUND);
+            }else{
+                return res.fromJSON(raw.Item.secret.S);
+            };     
+        } catch (e) {
+            throw e;
```

Comment:
If you're just going to re-throw the error, you may as well just omit the catch block :)